server/test/boardSettings.test.js

```diff
@@ -27,30 +27,39 @@ describe('Get board team members', () => {
       members: [
         {
           user: '22',
-          role: 'SUPER_ADMIN',
+          role: 'SUPER_ADMIN'
```

Comment:
@manishmz Committing line with removing commas add your name to git history and later on your name comes when we do git blame and if we are stuck in an issue it would be difficult to back track. Try not to commit these kinds of changes which involves syntactical refactoring.

Reply:
okay